### PR TITLE
Require libyuv by default.

### PR DESCRIPTION
**Pull request by vrabaud**
It can still be disabled with -DAVIF_LIBYUV=OFF.
Also bump libyuv.

### CODE REVIEW

File: .github/workflows/ci-unix-shared-installed.yml [hunk 1]
```diff
@@ -29,6 +29,9 @@ jobs:
     - name: Set GCC & G++ compiler (on Linux)
       if: runner.os == 'Linux'
       run: echo "CC=gcc-${{matrix.gcc}}" >> $GITHUB_ENV && echo "CXX=g++-${{matrix.gcc}}" >> $GITHUB_ENV
+    - name: Disable libyuv on macOS
+      if: runner.os == 'macOS'
+      run: echo "CMAKE_AVIF_FLAGS=\"-DAVIF_LIBYUV=OFF\""  >> $GITHUB_ENV
```
Comment (y-guyon) [inline .github/workflows/ci-unix-shared-installed.yml:34]: Add a comment mentioning why it is disabled on macOS

**Review by y-guyon — APPROVED** [review]

File: .github/workflows/ci-unix-shared-installed.yml (hunk 1 above)
Comment (vrabaud) [reply .github/workflows/ci-unix-shared-installed.yml]: Done

**Review by wantehchang — APPROVED** [review]
LGTM. Thanks. Could you link to b/325476398 in the commit message?

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,11 +48,14 @@ The changes are relative to the previous release, unless the baseline is specifi
...
   AVIF_COLOR_PRIMARIES_DCI_P3, AVIF_TRANSFER_CHARACTERISTICS_PQ.
 * Add avifResult enum entry AVIF_RESULT_INTERNAL_ERROR.
+* Have libyuv required by default (but it still can be disabled with
+  -DAVIF_LIBYUV=OFF).
 
 ### Changed since 1.0.0
 * Update aom.cmd: v3.8.1
 * Update dav1d.cmd: 1.4.0
 * Update libgav1.cmd: v0.19.0
+* Update libyuv.cmd: a6a2ec65
```
[the remark this answers is not in the document]
Comment (vrabaud) [reply CHANGELOG.md:58]: Done in https://github.com/AOMediaCodec/libavif/pull/2025

**Pull request merged**